envs/conda_build_config.yaml

```diff
@@ -139,7 +139,7 @@ libtiff:
 libtool:
   - 2.4.6
 lmdb:
-  - 0.9.22.*
+  - 0.9.29.*
```

Comment:
Maybe we just loosen this pin?

Reply:
Yes, it is needed. I had it in my local copy where I was also using changes for the protobuf refresh. Added the change here now.

Reply:
yes we can do so. Updating it.